apps/ledger-live-mobile/android/app/src/main/jni/MainComponentsRegistry.cpp

```diff
@@ -0,0 +1,48 @@
+#include "MainComponentsRegistry.h"
```

Comment:
This file isn't used yet as we are not activating the new RN architecture (Fabric + TurboModules)